frontends/bsd/src/screens/ballot_eject_screen.test.tsx

```diff
@@ -9,6 +9,14 @@ import { act } from 'react-dom/test-utils';
 import { renderInAppContext } from '../../test/render_in_app_context';
 import { BallotEjectScreen } from './ballot_eject_screen';
 
+jest.mock('@votingworks/types', () => {
+  return {
+    ...jest.requireActual('@votingworks/types'),
+    // mock election hash so snapshots don't change with every change to the election definition
+    getDisplayElectionHash: () => '0000000000',
+  };
+});
+
```

Comment:
### Election Hash Testing Strategy 5
Calculate the 10 character display hash in a helper function form `libs/ui` and mock it so that it does not change in every snapshot.

Reply:
Removed here.